traitsui/testing/tester/wx/helpers.py

```diff
@@ -316,3 +316,26 @@ def key_sequence_text_ctrl(control, interaction, delay):
     for char in interaction.sequence:
         wx.MilliSleep(delay)
         control.WriteText(char)
+
+
+def key_click_slider(control, interaction, delay):
+    valid_keys = {"Left", "Right", "Up", "Down", "Page Up", "Page Down"}
```

Comment:
I also realize something fun... on OSX, Up / Page Up increments the value. On another Linux machine of mine, Up / Page Up decrements the value. I did not notice that before.

This reminds me of the system setting to do with natural scrolling on trackpad / mouse wheel... some people think mouse wheel up should do a page down, where some people think a mouse wheel up should do a page up.  Perhaps this is yet another personal preference and we should just stick with Left and Right, which is unambiguous.

Reply:
This is when interacting with the UI manually I assume?  Also are you suggesting to remove "Up", "Page Up", etc from the list of valid keys, or just the tests (I read that as from the list of valid keys but wanted to confirm).  I had not thought of the case where "Up" would decrement.  I did like the ability to "Page Up"/"Page Down" for the difference is step/page step size but if it would be confusing to users it could be removed.  Could we also just document that "Up" corresponds to increment for the tester?

Reply:
Yes my comment was from manual testing.

> I did like the ability to "Page Up"/"Page Down" for the difference is step/page step size but if it would be confusing to users it could be removed. Could we also just document that "Up" corresponds to increment for the tester?

Yes I think documenting that "Up" corresponds to increment would also be reasonable!